### DISCUSSION TITLE
How can I remove unuseful apps/connection

### DISCUSSION THREAD

**Discussion by zaphod1966** [BimmerGestalt/AAIdrive#766] [Q&A]
Hi all...

Is it possible to remove (for me) unuseful apps?
![Screenshot_20240213-185855](https://github.com/BimmerGestalt/AAIdrive/assets/113337954/663b393b-c225-4443-88da-60d641fca9aa)

Thanks a lot...

**Comment by hufman** [comment]
You can swipe music applications off the side of that screen and they will become disabled and not show up in the car. You can even toggle them while connected to the car and they will update instantly in the car's media menu.

**Reply by zaphod1966** [reply]
So easy... Thanks
